core/dbt/jsonschemas.py

```diff
@@ -57,6 +58,10 @@ def error_path_to_string(error: jsonschema.ValidationError) -> str:
 
 
 def jsonschema_validate(schema: Dict[str, Any], json: Dict[str, Any], file_path: str) -> None:
+
+    if not os.environ.get("DBT_ENV_PRIVATE_RUN_JSONSCHEMA_VALIDATIONS"):
```

Comment:
Approving, as this is temporary, but in general I want to discourage checking env vars directly. They can be checked via the InvocationContext to provide a level of security and indirection.